--- ks/root/Makefile	2017-05-19 11:28:20.966814238 +0200
+++ vpp-17.0.4-ioam-anycast/build-root/Makefile	2017-05-31 19:03:40.738305659 +0200
@@ -196,9 +196,9 @@
 MARCH = $(strip $($(PLATFORM)_march))
 ifeq ($(MARCH),)
   ifneq ($(wildcard $(TOOL_INSTALL_DIR)/bin/$(TARGET)-gcc),)
-    TARGET_GCC = $(TOOL_INSTALL_DIR)/bin/$(TARGET)-gcc
+    TARGET_GCC ?= $(TOOL_INSTALL_DIR)/bin/$(TARGET)-gcc
   else ifneq ($(wildcard $(MU_BUILD_ROOT_DIR)/tools/bin/$(TARGET)-gcc),)
-    TARGET_GCC = $(MU_BUILD_ROOT_DIR)/tools/bin/$(TARGET)-gcc
+    TARGET_GCC ?= $(MU_BUILD_ROOT_DIR)/tools/bin/$(TARGET)-gcc
   endif
   ifneq ($(TARGET_GCC),)
     MARCH = $(shell $(TARGET_GCC) -Q --help=target -march=native | grep march | sed -e 's/.*march=[[:space:]]*//')
@@ -260,9 +260,9 @@
 
 ## BURT
 # we will deprecate INSTALL_DIR shortly for DFLT_INSTALL_DIR
-INSTALL_DIR = $(MU_BUILD_ROOT_DIR)/$(INSTALL_PREFIX)$(ARCH)
+INSTALL_DIR = $(TARGET_DIR)
 # DFLT_INSTALL_DIR used in platforms.mk for $(PLATFORM)_DESTDIR_BASE
-DFLT_INSTALL_DIR := $(MU_BUILD_ROOT_DIR)/$(INSTALL_PREFIX)$(ARCH)
+DFLT_INSTALL_DIR := $(TARGET_DIR)
 ## BURT
 
 PLATFORM_IMAGE_DIR = $(MU_BUILD_ROOT_DIR)/$(IMAGES_PREFIX)$(PLATFORM)
@@ -290,14 +290,13 @@
 
 PACKAGE_BUILD_DIR = \
   $(BUILD_DIR)/$(call package_build_dir_fn,$(PACKAGE))
-PACKAGE_INSTALL_DIR = \
-  $(call package_install_dir_fn,$(PACKAGE))
+PACKAGE_INSTALL_DIR = $(TARGET_DIR)
 
 # Tools (gcc, binutils, glibc...) are installed here
 TOOL_INSTALL_DIR = $(MU_BUILD_ROOT_DIR)/tools
 
 # Target specific tools go here e.g. mu-build/tools/ppc-mu-linux
-TARGET_TOOL_INSTALL_DIR = $(TOOL_INSTALL_DIR)/$(TARGET)
+TARGET_TOOL_INSTALL_DIR = $(TARGET_DIR)
 
 # Set BUILD_DEBUG to vx or x enable shell command tracing.
 BUILD_DEBUG =
@@ -491,12 +490,13 @@
 x86_64_libdir = $(BIARCH)
 native_libdir = $($(NATIVE_ARCH)_libdir)
 
-# lib or lib64 depending
-arch_lib_dir = lib$($(BASIC_ARCH)_libdir)
+# lib or lib64 depending - already defined earlier
+# arch_lib_dir = lib$($(BASIC_ARCH)_libdir)
+# buildroot dynamic libraries default location differ from the HOST_TOOLS
 
 # find dynamic linker as absolute path
-TOOL_INSTALL_LIB_DIR=$(TOOL_INSTALL_DIR)/$(TARGET)/$(arch_lib_dir)
-DYNAMIC_LINKER=${shell cd $(TOOL_INSTALL_LIB_DIR); echo ld*.so.*}
+TARGET_LIB_INSTALL_DIR=$(TARGET_TOOL_INSTALL_DIR)/$(arch_lib_dir)
+DYNAMIC_LINKER=${shell cd $(TARGET_LIB_INSTALL_DIR); echo ld*.so.*}
 
 # Pad dynamic linker & rpath so elftool will never have to change ELF section sizes.
 # Yes, this is a kludge.
@@ -507,8 +507,9 @@
 CROSS_LDFLAGS = $(strip $($(PLATFORM)_cross_ldflags))
 ifeq ($(CROSS_LDFLAGS),)
   CROSS_LDFLAGS =											\
-    -Wl,--dynamic-linker=$(lots_of_slashes_to_pad_names)$(TOOL_INSTALL_LIB_DIR)/$(DYNAMIC_LINKER)	\
-    -Wl,-rpath -Wl,$(lots_of_slashes_to_pad_names)$(TOOL_INSTALL_LIB_DIR)
+    -Wl,--dynamic-linker=/$(arch_lib_dir)/$(DYNAMIC_LINKER)	\
+    -Wl,-rpath -Wl,/$(arch_lib_dir) \
+    -Wl,-rpath -Wl,/usr/$(arch_lib_dir)
 endif
 
 cross_ldflags = $(if $(is_native)$(is_build_tool),,$(CROSS_LDFLAGS) )
@@ -706,11 +707,8 @@
 # Package install
 ######################################################################
 
+# package get's embedded inside fakeroot tree so, no partial removal available ;
 install_package =								\
-    : by default, for non-tools, remove any previously installed bits ;		\
-    $(if $(is_build_tool)$($(PACKAGE)_keep_instdir),				\
-         true,									\
-         rm -rf $(PACKAGE_INSTALL_DIR));					\
     mkdir -p $(PACKAGE_INSTALL_DIR) ;						\
     $(if $($(PACKAGE)_pre_install),$($(PACKAGE)_pre_install),true);		\
     $(if $($(PACKAGE)_install),							\
